File: 2455-average-value-of-even-numbers-that-are-divisible-by-three/2455-average-value-of-even-numbers-that-are-divisible-by-three.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var averageValue = function(nums) {
    let sum=0
    let count=0
    for(num of nums){
        if(num%2 == 0 && num % 3== 0){
            sum=sum+num
            count++
        }
    }
    if(sum>0){
        return Math.floor(sum/count)
    }else{
        return 0
    }
};

File: 2455-average-value-of-even-numbers-that-are-divisible-by-three/2455-average-value-of-even-numbers-that-are-divisible-by-three.go
func averageValue(nums []int) int {
    sum:=0
    count:=0
    for _,n:= range nums{
        if n%6==0{
            sum+=n
            count++
        }
    }
    if count==0{
            return 0
        }else{
            return sum/count
        }
}